Hello-World example flash error on Arduino Due

in folder RIOT/examples/hello-world
I executed:
make flash
Output:
Building application "hello-world" for "arduino-due" with MCU "sam3x8e".

"make" -C /home/abdur/Downloads/RIOT/boards/arduino-due
"make" -C /home/abdur/Downloads/RIOT/core
"make" -C /home/abdur/Downloads/RIOT/cpu/sam3x8e
"make" -C /home/abdur/Downloads/RIOT/cpu/cortex-m3_common
"make" -C /home/abdur/Downloads/RIOT/cpu/sam3x8e/periph
"make" -C /home/abdur/Downloads/RIOT/drivers
"make" -C /home/abdur/Downloads/RIOT/sys
"make" -C /home/abdur/Downloads/RIOT/sys/auto_init
   text	   data	    bss	    dec	    hex	filename
  11804	    116	   6572	  18492	   483c	/home/abdur/Downloads/RIOT/examples/hello-world/bin/arduino-due/hello-world.elf
/bin/stty

ERROR:
 Flash program stty -F /dev/ttyACM0 raw ispeed 1200 ospeed 1200 cs8 -cstopb ignpar eol 255 eof 255; /home/abdur/Downloads/RIOT/boards/arduino-due/dist/bossac not found. Aborting.
make: *** [flash] Error 1

I have checked the existance of bossac file and its there.

--- a/boards/arduino-due/Makefile.include
+++ b/boards/arduino-due/Makefile.include
@@ -11,20 +11,18 @@ export LINK = $(PREFIX)gcc
 export SIZE = $(PREFIX)size
 export OBJCOPY = $(PREFIX)objcopy
 export TERMPROG = $(RIOTBASE)/dist/tools/pyterm/pyterm
+export FLASHER = $(RIOTBOARD)/$(BOARD)/dist/flash.sh
 
 #define the flash-tool and default port depending on the host operating system
 OS := $(shell uname)
 ifeq ($(OS),Linux)
   PORT ?= /dev/ttyACM0
-  FLASHER = stty -F $(PORT) raw ispeed 1200 ospeed 1200 cs8 -cstopb ignpar eol 255 eof 255; $(RIOTBOARD)/$(BOARD)/dist/bossac
 else ifeq ($(OS),Darwin)
   PORT ?= $(shell ls -1 /dev/tty.usbmodem* | head -n 1)
-  FLASHER = stty -f $(PORT) raw ispeed 1200 ospeed 1200 cs8 -cstopb ignpar eol 255 eof 255; $(RIOTBOARD)/$(BOARD)/dist/bossac_osx
 else
   $(info CAUTION: No flash tool for your host system found!)
   # TODO: add support for windows as host platform
 endif
-export FLASHER
 export PORT
 
 # define build specific options
@@ -36,7 +34,6 @@ export LINKFLAGS += -g3 -ggdb -std=gnu99 $(CPU_USAGE) $(FPU_USAGE) -mlittle-endi
 # linkerscript specified in cpu/Makefile.include
 export LINKFLAGS += -T$(LINKERSCRIPT)
 export OFLAGS += -O binary
-export FFLAGS += -R -e -w -v -b bin/$(BOARD)/$(APPLICATION).hex
 export TERMFLAGS += -p "$(PORT)"
 
 # use the nano-specs of the NewLib when available

--- a/boards/spark-core/Makefile.include
+++ b/boards/spark-core/Makefile.include
@@ -17,7 +17,7 @@ export LINK = $(PREFIX)gcc
 export SIZE = $(PREFIX)size
 export OBJCOPY = $(PREFIX)objcopy
 export TERMPROG = $(RIOTBASE)/dist/tools/pyterm/pyterm
-export FLASHER = $(OBJCOPY) -O binary $(ELFFILE) $(BINFILE) ; dfu-util -d 1d50:607f -a 0 -s 0x08005000:leave -D "$(BINFILE)"
+export FLASHER = dfu-util
 export DEBUGGER = # spark core has no debugger
 export RESET = # dfu-util has no support for resetting the device
 
@@ -30,9 +30,8 @@ export ASFLAGS += -ggdb -g3 $(CPU_USAGE) $(FPU_USAGE) -mlittle-endian
 export LINKFLAGS += -ggdb -g3 -std=gnu99 $(CPU_USAGE) $(FPU_USAGE) -mlittle-endian -static -lgcc -mthumb -mno-thumb-interwork -nostartfiles
 # $(LINKERSCRIPT) is specified in cpu/Makefile.include
 export LINKFLAGS += -T$(LINKERSCRIPT)
-export OFLAGS = -O ihex
-export FFLAGS = $(HEXFILE)
-export DEBUGGER_FLAGS = $(RIOTBOARD)/$(BOARD)/dist/gdb.conf $(ELFFILE)
+export OFLAGS = -O binary
+export FFLAGS = -d 1d50:607f -a 0 -s 0x08005000:leave -D "$(HEXFILE)"
 export TERMFLAGS = -p $(PORT)
 
 # unwanted (CXXUWFLAGS) and extra (CXXEXFLAGS) flags for c++

